### ISSUE TITLE
Add client.enableEmergency(); functionality to trigger emergency cutout

### ISSUE THREAD

**Issue by ColinCreamer** [felixge/node-ar-drone#123]
It would be great if this API provided an easy way to trigger an emergency cutout while flying, to make it dead drop from the air. Similar to how you can easily clear an emergency via client.disableEmergency(); having a client.enableEmergency(); function as well, would allow you to just as easily trigger an emergency.

**Comment by wiseman** [comment]
Good idea!  It might be a little while before I have a chance to do this, do you want to give it a try?

Take a look at how the [`disableEmergency` function](https://github.com/felixge/node-ar-drone/blob/master/lib/Client.js#L189) is implemented, including the logic in [`_processNavdata`](https://github.com/felixge/node-ar-drone/blob/master/lib/Client.js#L109).

Page 35 of the [AR.Drone Developer Guide](http://www.msh-tools.com/ardrone/ARDrone_Developer_Guide.pdf) describes how to send the "emergency order"; It will be very similar to how we currently send the "disable emergency" command.
